### ISSUE TITLE
Coding Standard update: Lines should not have trailing whitespace

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2216]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#2216|
|Type|enhancement|
|Created|2006-11-04 07:02:30Z|

I think we are regularly deleting trailing whitespace. We should update the policy to match practice.

<details><summary>Searchable metadata</summary>

```
trac-id__2216 2216
type__enhancement enhancement
reporter__jml jml
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__policy documentation, policy
time__1162623750000000 1162623750000000
changetime__1190967011000000 1190967011000000
version__None None
owner__therve therve
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve removed owner|
|-|-|

Let's kick this out, I want to progress on coding standard update. The following patch should do the trick:

```
Index: doc/development/policy/coding-standard.xhtml
===================================================================
--- doc/development/policy/coding-standard.xhtml        (revision 21320)
+++ doc/development/policy/coding-standard.xhtml        (working copy)
@@ -72,7 +72,8 @@
     is preferred that every block appear on a new line, so that
     control structure indentation is always visible.</p>

-    <p>Lines are flowed at 79 columns.</p>
+    <p>Lines are flowed at 79 columns. They must not have trailing
+    whitespaces.</p>

     <h2>Modules</h2>

```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @therve|
|-|-|

Looks good.  Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set status to `closed`|
|-|-|

(In [21321]) Update coding standard: lines must not have trailing whitespaces.

Author: therve
Reviewer: exarkun
Fixes [#2216](https://github.com/twisted/twisted/issues/2216)
